lib/eslint.js

```diff
@@ -77,7 +88,7 @@ function run(console, runOptions) {
           },
         },
       },
-      remediation_points: checks.remediationPoints(checkName, message, cli.getConfigForFile(path)),
+      remediation_points: checks.remediationPoints(checkName, message, cli.calculateConfigForFile(path)),
```

Comment:
I guess this is just a name change on eslint internals api, from `getConfigForFile` to `calculateConfigFile` ? Didn't see a note about that on eslint's release notes, can you explain why is this necessary ?

Reply:
Ok found the explanation here https://eslint.org/docs/user-guide/migrating-to-8.0.0#remove-cliengine